azext_iot/_params.py

```diff
@@ -772,6 +772,60 @@ def load_arguments(self, _):
             help="Maximum number of configurations to return. By default all configurations are returned.",
         )
 
+    with self.argument_context("iot hub state") as context:
+        context.argument(
+            "filename",
+            options_list=["--filename", "-f"],
+            help="The path to the file where the hub and device information will be stored."
+        )
+        context.argument(
+            "replace",
+            options_list=["--replace", "-r"],
+            help="If this flag is set, then the import will overwrite all configurations and devices in the destination hub."
```

Comment:
the import will? you can just say:
If this flag is set, then all configurations and devices in the destination hub will be overwritten.

Although this is passive tense. We many need to consult word. If you want to make it active - then use command instead of import

Reply:
Right - I copied this over and forgot to make it command-neutral!